### ISSUE TITLE
Me cierra la sesión de Google de manera automática

### ISSUE THREAD

**Issue by manzana22-app** [midudev/aprende-javascript-problemas#375]
No me da señal de el porque pero me cierra la sesión sin aviso, sin razón solo lo hace, no se a que se deberá pero seria bueno que lo revisen

**Comment by PsychoNoobie** [comment]
A mi tambien me sucede.

**Comment by AlejandroAlea** [comment]
Sigue pasando

**Comment by DeeDstrox** [comment]
sigue pasando

**Comment by Dannaardil** [comment]
a mi tambien  me pasa 
que hago

**Comment by thejasondev** [comment]
También me pasa, alguna solución midu?

**Comment by MaxGon19** [comment]
a mi igual y lo he probado tanto en google como brave

**Comment by danifer18** [comment]
hoy 19/dic/2024 me pasó. 
Ingreso y luego me cierra la sesión, y se queda así todo el tiempo... ¿Alguna solución?
Nota: sucede en el apartado: ¿Qué son los Array en JavaScript? Ya que seleccioné el siguiente tema de arrays y no sucede.

**Comment by LeoSan023** [comment]
Efectivamente, al día de **hoy 02-Enero-2025** sigue pasando este incidente tanto con Git como con Google. En mi caso intenté con Google Chrome y Brave y el comportamiento es el mismo, al cabo de apróximadamente 10 seg se recarga y se deslogea automáticamente y se repite lo mismo de forma indefinida.
